cardinal/testutils/test_utils.go

```diff
@@ -136,14 +145,18 @@ func uniqueSignature() *sign.SignedPayload {
 	nonce++
 	// We only verify signatures when hitting the HTTP server, and in tests we're likely just adding transactions
 	// directly to the World queue. It's OK if the signature does not match the payload.
-	sig, err := sign.NewSignedPayload(privateKey, "some-persona-tag", "namespace", nonce, `{"some":"data"}`)
+	sig, err := sign.NewSignedPayload(privateKey, personaTagName, "namespace", nonce, `{"some":"data"}`)
 	if err != nil {
 		panic(err)
 	}
 	return sig
 }
 
-func AddTransactionToWorldByAnyTransaction(world *cardinal.World, cardinalTx cardinal.AnyTransaction, value any) {
+func AddTransactionToWorldByAnyTransaction(
+	world *cardinal.World,
+	cardinalTx cardinal.AnyTransaction,
+	value any,
+	signedPayload ...*sign.SignedPayload) {
```

Comment:
Same as above. In this case, the single AddTransactionToWorldbyAnyTransaction should just have a required signature parameter; it's easy to make that signature with the above UniqueSignature method.